fpdf/image_parsing.py

```diff
@@ -171,8 +174,12 @@ def _to_zdata(img, remove_slice=None, select_slice=None):
     if select_slice:
         data = data[select_slice]
     # Left-padding every row with a single zero:
-    channels_count = len(data) // (img.size[0] * img.size[1])
-    loop_incr = img.size[0] * channels_count + 1
+    if img.mode == '1':
+        import math
```

Comment:
Please move this `import` at the top of this file

Reply:
Done